src/components/ButtonPanel.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import Button from './Button';
+
+function ButtonPanel() {
+    return (
+        <div>
+            <div className = "groupone">
+                <Button name = "AC"/>
+                <Button name = "+/-"/>
+                <Button name="%" />
+                <Button name="÷" />
+            </div>
+            <div className = "grouptwo">
+                <Button name = "7"/>
+                <Button name = "8"/>
+                <Button name="9" />
+                <Button name="X" />
+            </div>
+            <div className = "groupthree">
+                <Button name = "4"/>
+                <Button name = "5"/>
+                <Button name="6" />
+                <Button name="-" />
+            </div>
+            <div className = "groupfour">
+                <Button name = "1"/>
+                <Button name = "2"/>
+                <Button name="3" />
+                <Button name="+" />
+            </div>
+            <div className = "groupfive">
+                <Button name = "0"/>
+                <Button name = "."/>
+                <Button name="=" />
+              
+            </div>
+        </div>
+    )
+
+}
+
+export default ButtonPanel;
```

Comment:
Also in here, if you make a button array with all the symbols and map through these buttons to render a Button component, it would be a more elegant solution. You can think about it. :100: